lib/hello_world/auth/auth0_config.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HelloWorld.Auth.Auth0Config do
+  def auth0_audience, do: Application.get_env(:hello_world, :auth0_audience)
+
+  def auth0_domain do
```

Comment:
@manusajith I think we should name this differently in order to avoid some misunderstandings, something like `auth0_base_url` perhaps?